src/Midgard/CreatePHP/RdfChainableMapperInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @copyright CONTENT CONTROL GmbH, http://www.contentcontrol-berlin.de
+ * @author CONTENT CONTROL GmbH, http://www.contentcontrol-berlin.de
+ * @license Dual licensed under the MIT (MIT-LICENSE.txt) and LGPL (LGPL-LICENSE.txt) licenses.
+ * @package Midgard.CreatePHP
+ */
+
+namespace Midgard\CreatePHP;
+
+use Midgard\CreatePHP\Type\TypeInterface;
+
+/**
+ * Map from CreatePHP to your domain objects
+ *
+ * You can have a mapper per type or a generic mapper that handles all types.
```

Comment:
please update this description to explain what the chainable mapper is. the general description of what a mapper is can stay in RdfMapperInterface